extensions/samples/electron-webpack/src/AuthProvider.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { shell } from "electron";
+import {
+    PublicClientApplication,
+    LogLevel,
+    AccountInfo,
+    SilentFlowRequest,
+    AuthenticationResult,
+    InteractiveRequest,
+} from "@azure/msal-node";
+import {
+    DataProtectionScope,
+    Environment,
+    PersistenceCreator,
+    PersistenceCachePlugin,
+} from "@azure/msal-node-extensions";
+import path from "path";
+
+export default class AuthProvider {
+    private clientApplication: PublicClientApplication;
+    private account: AccountInfo;
+
+    constructor() {}
+
+    static async build(authConfig: any): Promise<any> {
```

Comment:
shouldn't `authConfig` here be of type `Configuration`?

Reply:
Done